### ISSUE TITLE
[FEATURE] Onboarding regarding the main concepts

### ISSUE THREAD

**Issue by IvoPereira** [CharlieBrownCharacter/feedback#14]
**Is your feature request related to a problem? Please describe.**
A first-time user might not be be sure of what exactly are the advantages or what exactly is the concept of a group, meetup and talk and what each one of them involves.

**Describe the solution you'd like**
After logging in first time, a modal overlay (or a modal near each concept in the menu - like group, meetup or talk) should appear to drive the user and let him know what each part is responsible for. For instance, "Groups would let people engage together in a group conversation and allow meetup talks".

These modals should however only appear at the first-time, or at least provide a checkbox whether the user wants to be remembered or not.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by CharlieBrownCharacter** [comment]
@IvoPereira Yeah, we have thought about it and we have a card for it. We are still deciding on how exactly should work. Your suggestion seems pretty straight forward. Ty
